### ISSUE TITLE
minor display bug: shifting right margin when mousing over graph overview

### ISSUE THREAD

**Issue by gglusman** [NCATSTranslator/Feedback#476]
While outside the graph overview:
![image](https://github.com/NCATSTranslator/Feedback/assets/3977332/3c33eba0-aedd-42bd-83cb-c05a0a63e54e)

While hovering over it:
![image](https://github.com/NCATSTranslator/Feedback/assets/3977332/c0ec7847-14da-4af3-992e-ad7976067b35)

**Comment by gglusman** [comment]
Hmm, still happening on Chrome on Apple computer A, but not on two other Apple computers B and C (also Chrome)...
Tested with Safari on computer A - same jumping effect as with Chrome.

I note that on computers B and C, the blue ribbon at the top always has the gap at the right window edge, looking as in the 2nd screenshot.

All three computers running Chrome 115.0.5790.170. Computers A and B have arm64 version of Chrome, computer C uses a different build.

**Comment by dnsmith124** [comment]
Thank you for the incredibly detailed platform breakdown @gglusman! I seriously can't express how helpful that is. 

I have a hunch that this is related to a browser setting of some sort, but I'm not certain. After the relay I'll dig into this more thoroughly.

**Comment by gglusman** [comment]
Just tested on a 4th computer, no issue.
What's puzzling to me is that this happens just on one computer, but both on Chrome and Safari...

**Comment by dnsmith124** [comment]
This appears to be an issue with the scrollbar setting in a given user's browser/operating system, if it's set to be an _overlay_ there's no issue, but if it's set to be _in the page_ certain functionalities in the UI that cause the scrolling to change (such as holding the Z key to scroll a graph, or using the navigator window) can cause the _in page_ scrollbar to disappear, which causes the shift.

This is on the UI team's radar, and will require a more universal overhaul of the scrolling functionality within the UI that will apply to all users regardless of browser/OS settings.

**Comment by gglusman** [comment]
Observing the same shift-and-jump effect on another computer D, Intel i5 Mac, 2017 model, running Ventura. Chrome 121.0.6167.139.
On this computer, though, it looks fine with Safari.

(And yes, it's still happening with computer A as above.)

**Comment by sstemann** [comment]
@dnsmith124 should this be retested after the facelift?  if so, can we tag if for Guppy or Hammerhead?

**Comment by dnsmith124** [comment]
@sstemann yes! A fix for this will be included in the facelift, I'll go ahead and tag it for Guppy.
